package.json

```diff
@@ -4,6 +4,8 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "@elastic/elasticsearch": "^7.3.0",
+    "elasticsearch": "^16.3.0",
```

Comment:
make sure you are making commits for each package you use

Reply:
10-4 my bad